<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulerGUIClass</class>
 <widget class="QMainWindow" name="SchedulerGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_schedule"/>
    <addaction name="action_Open_Schedule"/>
    <addaction name="action_Save_schedule"/>
    <addaction name="action_Save_schedule_as"/>
    <addaction name="action_Close_schedule"/>
    <addaction name="action_Save_task_list"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Schedule_Settings"/>
    <addaction name="action_Load_settings"/>
    <addaction name="actionLoad_default_settings"/>
    <addaction name="action_Save_settings"/>
    <addaction name="action_Save_default_settings"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="action_Create_initial_schedule"/>
    <addaction name="action_Optimize_schedule"/>
    <addaction name="action_Balance_schedule"/>
    <addaction name="action_Assign_resources"/>
    <addaction name="action_Align_left"/>
    <addaction name="actionPublish_schedule"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add_task"/>
    <addaction name="action_Delete_task"/>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="action_Find"/>
    <addaction name="action_Thrashcan"/>
    <addaction name="action_Move_task"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuGraphical_schedule_view">
     <property name="title">
      <string>&amp;Graphical schedule view</string>
     </property>
     <addaction name="actionGraphicalViewShow"/>
     <addaction name="actionResource_view"/>
     <addaction name="actionTask_view"/>
    </widget>
    <widget class="QMenu" name="menuTable_view">
     <property name="title">
      <string>&amp;Table view</string>
     </property>
     <addaction name="actionTableViewShow"/>
     <addaction name="actionSelect_colums_to_view"/>
    </widget>
    <addaction name="menuGraphical_schedule_view"/>
    <addaction name="menuTable_view"/>
    <addaction name="actionHorizontal_view"/>
    <addaction name="actionVertical_view"/>
    <addaction name="action_Statistics"/>
   </widget>
   <widget class="QMenu" name="menuSAS">
    <property name="title">
     <string>SAS</string>
    </property>
    <addaction name="action_DownloadSASSchedule"/>
    <addaction name="action_SyncSASSchedule"/>
    <addaction name="action_Compare_differences"/>
    <addaction name="actionCheck_SAS_status"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnect_to_Data_monitor"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Actions"/>
   <addaction name="menuSAS"/>
   <addaction name="menuConnect"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Close_schedule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close schedule</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Optimize_schedule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Optimize schedule</string>
   </property>
   <property name="toolTip">
    <string>Optimize schedule (Ctrl + O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Balance_schedule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Balance schedule</string>
   </property>
   <property name="toolTip">
    <string>Balance schedule</string>
   </property>
  </action>
  <action name="action_Open_task_list">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open task list</string>
   </property>
  </action>
  <action name="action_Create_initial_schedule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Create initial schedule</string>
   </property>
   <property name="toolTip">
    <string>Create initial schedule</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit (Ctrl + Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Save_task_list">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export task list</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo (Ctrl + Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo (Ctrl + Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_Save_schedule_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save schedule as</string>
   </property>
  </action>
  <action name="action_Open_Schedule">
   <property name="text">
    <string>&amp;Open schedule</string>
   </property>
  </action>
  <action name="action_Save_schedule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save schedule</string>
   </property>
   <property name="toolTip">
    <string>Save schedule (Ctrl + S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="close">
   <property name="icon">
    <iconset>
     <normaloff>images/close16x16.png</normaloff>images/close16x16.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_Schedule_Settings">
   <property name="text">
    <string>&amp;Schedule settings</string>
   </property>
  </action>
  <action name="action_Save_settings">
   <property name="text">
    <string>S&amp;ave settings</string>
   </property>
  </action>
  <action name="action_Load_settings">
   <property name="text">
    <string>&amp;Load settings</string>
   </property>
  </action>
  <action name="action_Save_default_settings">
   <property name="text">
    <string>Save as &amp;default settings</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_New_schedule">
   <property name="text">
    <string>&amp;New schedule</string>
   </property>
  </action>
  <action name="actionLoad_default_settings">
   <property name="text">
    <string>Load default settings</string>
   </property>
  </action>
  <action name="actionShow_Status_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status bar</string>
   </property>
  </action>
  <action name="actionSelect_colums_to_view">
   <property name="text">
    <string>Select colums to view</string>
   </property>
  </action>
  <action name="actionResource_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resource view</string>
   </property>
  </action>
  <action name="actionTask_view">
   <property name="text">
    <string>Task view</string>
   </property>
  </action>
  <action name="actionGraphicalViewShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionTableViewShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionHorizontal_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Horizontal view</string>
   </property>
  </action>
  <action name="actionVertical_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Vertical view</string>
   </property>
  </action>
  <action name="action_Align_left">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Align left</string>
   </property>
  </action>
  <action name="action_Add_task">
   <property name="text">
    <string>&amp;Add task</string>
   </property>
  </action>
  <action name="action_Delete_task">
   <property name="text">
    <string>&amp;Delete task(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="action_DownloadSASSchedule">
   <property name="text">
    <string>&amp;Download running schedule from SAS</string>
   </property>
  </action>
  <action name="action_SyncSASSchedule">
   <property name="text">
    <string>&amp;Synchronize schedule with SAS</string>
   </property>
  </action>
  <action name="action_Compare_differences">
   <property name="text">
    <string>&amp;Compare differences</string>
   </property>
  </action>
  <action name="action_Statistics">
   <property name="text">
    <string>&amp;Statistics</string>
   </property>
  </action>
  <action name="actionPublish_schedule">
   <property name="text">
    <string>&amp;Publish schedule</string>
   </property>
  </action>
  <action name="action_Thrashcan">
   <property name="text">
    <string>&amp;Thrashcan</string>
   </property>
  </action>
  <action name="actionCheck_SAS_status">
   <property name="text">
    <string>Check SAS status</string>
   </property>
  </action>
  <action name="action_Assign_resources">
   <property name="text">
    <string>Assign resources</string>
   </property>
  </action>
  <action name="actionConnect_to_Data_monitor">
   <property name="text">
    <string>Connect to Data Monitor</string>
   </property>
  </action>
  <action name="actionCleanup_Storage">
   <property name="text">
    <string>Cleanup &amp;Storage</string>
   </property>
  </action>
  <action name="action_Move_task">
   <property name="text">
    <string>&amp;Move task(s)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
